libyul/optimiser/VarDeclInitializer.cpp

```diff
@@ -47,11 +47,11 @@ void VarDeclInitializer::operator()(Block& _block)
 			else
 			{
 				OptionalStatements ret{vector<Statement>{}};
-				langutil::SourceLocation loc{std::move(_varDecl.location)};
+				auto debugData = _varDecl.debugData;
```

Comment:
Can you use an explicit type here so we can be sure that the value is copied?

Reply:
Not needed, just removed the line